docs/content/docs/install/settings.md

```diff
@@ -261,6 +261,15 @@ A few settings are available to configure this feature:
   If it find any strings matching the values of secrets attached to the
   PipelineRun it will replace it with the placeholder `******`
 
+* `error-log-snippet-number-of-lines`
+
+  default: `3`
```

Comment:
Same comment here about the documentation for `error-log-snippet` being now out of date, lines 252 through 262